### DISCUSSION TITLE
Obfuscate tool

### DISCUSSION THREAD

**Discussion by inkong** [airsdk/Adobe-Runtime-Support#3138] [Q&A]
Hello, Is there any tool that we can still use to obfuscate flex codes? Most of the tools were no longer supported. I need for both windows and mac. Appreciate the help.

**Comment by Shaunmax** [comment]
Have you tried secureswf ?

**Reply by inkong** [reply]
I tried accessing the website but I can't seem to download the apps. I'm not sure if they are still maintaining it.

**Reply by Shaunmax** [reply]
then, just search for the same on the internet, and you should get one.

**Reply by inkong** [reply]
I actually have tried looking at irrFuscator, DoSWF, SWF Encrypt but none of them works. Not sure if there still a tool that is working? Or anybody have any info and was able to use it recently?
